sous-chef/tests/unit_tests/recipe_book/test_read_recipe_book.py

```diff
@@ -519,6 +519,60 @@ def test__select_random_recipe_weighted_by_rating_removes_long_cooking_time(
             ).squeeze(),
         )
 
+    @staticmethod
+    @pytest.mark.parametrize(
+        "method,item_type",
+        [
+            ("get_random_recipe_by_category", "categories"),
+            ("get_random_recipe_by_tag", "tags"),
+        ],
+    )
+    @pytest.mark.parametrize(
+        "recipe_title,recipe_rating",
+        [
+            ("Excluded by rating", 2),
+            ("Not excluded by rating", 5),
+        ],
+    )
+    def test__select_random_recipe_weighted_by_rating_removes_low_rating(
```

Comment:
Could we add a similar test for the filter one?

Reply:
yep, good idea